advisories/2.2.0/BRSA-wyuvthdr.toml

```diff
@@ -0,0 +1,16 @@
+[advisory]
+id = "BRSA-wyuvthdr"
+title = "soci-snapshotter CVE-2024-24788"
```

Comment:
This is a Go-related vulnerability. Did it ever affect the soci-snapshotter package shipped in the core kit?

Reply:
> This is a Go-related vulnerability. Did it ever affect the soci-snapshotter package shipped in the core kit?

Oh, I see - it also required a `golang.org/x/net` update on the SOCI side. So this is fine.